README.md

```diff
@@ -161,6 +161,10 @@ The following assumptions are made in utilizing this project:
 ## Contributing to SIT
   * Please create a pull-request (preferably referring to an issue) with a single, meaningful commit message stating what you are accomplishing.
   * Add unit tests to new code
+    Before running unit tests, install the necessary requirements:
+    ```python
+    pip install -r /path/to/sit/tests/requirements.txt
```

Comment:
This is unnecessary. Should handle installing requirements via setup.py

![Image of Rick](http://assets.rollingstone.com/assets/2015/article/how-rick-and-morty-became-one-of-tvs-weirdest-hit-shows-20150722/203443/medium_rect/1437493748/720x405-RAM_S2_Ep205.jpg)